cg/cli/deliver/base.py

```diff
@@ -133,60 +51,56 @@ def rsync(context: CGConfig, ticket: str, dry_run: bool):
     )
 
 
-@deliver.command(name="concatenate")
-@DRY_RUN
-@TICKET_ID_ARG
-@click.pass_context
-def concatenate(context: click.Context, ticket: str, dry_run: bool):
-    """The fastq files in the folder generated using "cg deliver analysis"
-    will be concatenated into one forward and one reverse fastq file
+@deliver.command(name="case")
+@DELIVERY_TYPE
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="Deliver the files for a specific case",
+)
+@click.pass_obj
+def deliver_analysis(
+    context: CGConfig,
+    case_id: str,
+    delivery_type: list[str],
+):
     """
-    cg_context: CGConfig = context.obj
-    deliver_ticket_api = DeliverTicketAPI(config=cg_context)
-    deliver_ticket_api.concatenate_fastq_files(ticket=ticket, dry_run=dry_run)
+    Deliver all case files based on delivery type to the customer inbox on the HPC
+    """
+    inbox: str = context.delivery_path
+    service_builder = DeliveryServiceFactory(
+        store=context.status_db, hk_api=context.housekeeper_api
+    )
+    case: Case = context.status_db.get_case_by_internal_id(internal_id=case_id)
+    for delivery in delivery_type:
```

Comment:
If they are ok with not giving workflows as delivery types we can skip this loop!

Reply:
Yes, I removed it and also the option to pass delivery type. As it is set on the case